elastic/kibana
# [ResponseOps][Alerting] Elasticsearch Query Rule doesn't have 'dark mode' view for query

Resolves [#136900](https://github.com/elastic/kibana/issues/136900)

## Summary

Update the Elasticsearch Query Rule editor to use CodeEditor, which supports light and dark mode.
**Before:**
<img width="579" alt="Screen Shot 2022-08-11 at 11 28 25 AM" src="https://user-images.githubusercontent.com/109488926/184170989-2b661bcc-b4d5-4722-8fd9-b76f2b57bb94.png">

**After:**
<img width="472" alt="Screen Shot 2022-08-11 at 11 20 21 AM" src="https://user-images.githubusercontent.com/109488926/184170149-5d120c9e-b80d-42a1-8f85-832bc701250f.png">

_(light mode)_
<img width="476" alt="Screen Shot 2022-08-11 at 11 19 42 AM" src="https://user-images.githubusercontent.com/109488926/184170162-fdd30d38-ea55-4314-ae77-01abf2da3d34.png">

### Checklist


### To verify

- Enable dark mode in the [advanced settings](http://localhost:5601/app/management/kibana/settings)
- Create an Elasticsearch Query Rule and choose Query DSL. Verify that the editor is also in dark mode.
- Verify that you can create the rule and that it works as expected
